src/merge_tools.rs

```diff
@@ -277,17 +276,13 @@ pub fn run_mergetool(
     Ok(tree_builder.write_tree())
 }
 
-fn interpolate_mergetool_filename_patterns(
-    merge_args: &[String],
-    paths: &HashMap<&str, String>,
-) -> Vec<String> {
-    merge_args
-        .iter()
+fn interpolate_patterns(args: &[String], patterns: &HashMap<&str, String>) -> Vec<String> {
```

Comment:
Optional: perhaps `interpolate_dollar_patterns`?

Reply:
Or `interpolate_variables`?

I don't want to be specific about the syntax because this is the only substitution rule for merge_tools.